test/unit/specs/components/transactions/LiTransaction.spec.js

```diff
@@ -25,14 +25,14 @@ describe(`LiTransaction`, () => {
   })
 
   it(`Should print the hour only if the same day`, () => {
-    expect(LiTransaction.computed.date({ time: Date.now() })).toEqual(
-      `00:00:42`
+    expect(LiTransaction.computed.date({ time: new Date(420000).toISOString() })).toEqual(
+      `00:07:00`
     )
   })
 
   it(`Should print the datetime if we are in a different day`, () => {
     expect(
-      LiTransaction.computed.date({ time: Date.now() + 999999999 })
-    ).toEqual(`Jan 12th 1970 13:47:21`)
+      LiTransaction.computed.date({ time: new Date(4200000 + 999999999).toISOString() })
```

Comment:
let's user Date.new() if we change the date.now(function to another time all this hardcoded 4200000) will break

Reply:
`Date.new is not a function`

Reply:
and `new Date()` defaults to `Date.now()`

Reply:
this is about not hardcoding `4200000` in the test but using `Date.now()` so the tests don't break if we change the value `4200000` in the `Date.now` mock.

Reply:
yeah if you look into `test/unit/helpers/fixed_time.js` there is the mock

Reply:
only `Date.now()` is mocked

Reply:
My  bad, that's a typo: `Date.new()`